packages/testing/src/api/index.ts

```diff
@@ -102,7 +102,7 @@ function initializeApp(
       timestampsInSnapshots: true
     });
   }
-/**
+  /**
```

Comment:
I'm not sure why this changed?  It looks misaligned to me...

Reply:
It's the prettifier. Now it's aligned with the function it's commenting on. not sure how it was not caught before.